packages/server/dataloader/RethinkDBCache.ts

```diff
@@ -21,22 +21,21 @@ export default class RethinkDBCache {
     fetches.forEach((fetch) => {
       const {table, id} = fetch
       idsByTable[table] = idsByTable[table] || []
-      idsByTable[table].push(id)
+      idsByTable[table]!.push(id)
```

Comment:
+1 create a temp variable:

```javascript
const ids = idsByTable[table] || []
ids.push(id)
```

Reply:
See above, this wouldn't modify `idsByTable` which is the goal here.